backend/src/apiserver/config/sample_config.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "name": "[Demo] Unified DSL - Taxi Tip Prediction Model Trainer",
```

Comment:
Could you change the name to include TFX, currently it is not clear that this is a TFX based pipeline. 
for example could change it to something like : " [Demo] TFX Unified DSL - Taxi Tip Prediction Model Trainer"